examples/platform/silabs/efr32/rs911x/hal/rsi_hal_mcu_ioports.c

```diff
@@ -61,11 +61,21 @@ void rsi_hal_config_gpio(uint8_t gpio_number, uint8_t mode, uint8_t value)
 
     CMU_ClockEnable(cmuClock_GPIO, true);
 
-    // WFX_RSI_LOG ("RSI: CFG GPIO: 0x%x", gpio_number);
     switch (gpio_number)
     {
+    case RSI_HAL_SLEEP_CONFIRM_PIN:
+        GPIO_PinModeSet(WFX_SLEEP_CONFIRM_PIN.port, WFX_SLEEP_CONFIRM_PIN.pin, gpioModeWiredOrPullDown, 1);
+        break;
+#ifndef LOGGING_STATS
+    case RSI_HAL_WAKEUP_INDICATION_PIN:
+        GPIO_PinModeSet(WAKE_INDICATOR_PIN.port, WAKE_INDICATOR_PIN.pin, gpioModeWiredOrPullDown, 0);
+        break;
+#endif
     case RSI_HAL_RESET_PIN:
-        GPIO_PinModeSet(WFX_RESET_PIN.port, WFX_RESET_PIN.pin, gpioModePushPull, PINOUT_SET);
+        GPIO_PinModeSet(WFX_RESET_PIN.port, WFX_RESET_PIN.pin, gpioModePushPull, 1);
+        break;
+    case RSI_HAL_LP_SLEEP_CONFIRM_PIN:
+        GPIO_PinModeSet(WFX_SLEEP_CONFIRM_PIN.port, WFX_SLEEP_CONFIRM_PIN.pin, gpioModeWiredOrPullDown, 1);
```

Comment:
This switch-case is identic to `RSI_HAL_SLEEP_CONFIRM_PIN`. Can you refactor the switch case to something like :

```
case RSI_HAL_SLEEP_CONFIRM_PIN:
case RSI_HAL_LP_SLEEP_CONFIRM_PIN:
        GPIO_PinModeSet(WFX_SLEEP_CONFIRM_PIN.port, WFX_SLEEP_CONFIRM_PIN.pin, gpioModeWiredOrPullDown, 1);
        break;